python/lsst/ctrl/mpexec/cli/cmd/commands.py

```diff
@@ -126,8 +138,17 @@ def qgraph(ctx: click.Context, **kwargs: Any) -> None:
 def run(ctx: click.Context, **kwargs: Any) -> None:
     """Build and execute pipeline and quantum graph."""
     kwargs = _collectActions(ctx, **kwargs)
-    pipeline = script.build(**kwargs)
-    qgraph = script.qgraph(pipelineObj=pipeline, **kwargs)
+    show = ShowInfo(kwargs.pop("show", []))
+    pipeline = script.build(**kwargs, show=show)
+    if show.handled and not show.unhandled:
+        # The show option was given and all options were processed.
```

Comment:
I wonder if we should print this to the user so they aren't confused.

Reply:
Hmm, if the output from e.g. `dump-config` is redirected to a file, it may also confuse something.

Reply:
We can  print to stderr?

Reply:
stderr is probably better than stdout, though we should probably ask users about what they want. I myself almost never use `pipetask`, so my opinion probably should not count.